KartingCoach/Model/LapTime.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  LapTime.swift
+//  KartingCoach
+//
+//  Created by Jakub Olejník on 01/09/2017.
+//
+
+import Foundation
+
+struct LapTime {
+    static var zero: LapTime { return LapTime(duration: 0) }
+    
+    let minutes: UInt
+    let seconds: UInt
+    let miliseconds: UInt
+    
+    var duration: UInt {
```

Comment:
❔ proc `UInt`? Jdes proti konvencim a nic ti to neprinese

Reply:
To vim, ale nechci resit to ze se tam nekde strci zaporna dylka.

Reply:
Moc nechápu, jak bys tam dostal zápornou délku 😄 ale i tak se mi to nelíbí

Reply:
`LapTime(duration: -1)`

Reply:
to bych spíš ošetřil v `init`u a napsal na to test než abych strkal `UInt`y tam, kde být nemusejí / nemají

Reply:
Přijde mi zas lepší ať to ošetří kompilátor než abych se s tim patlal já

Reply:
dobře a co kompromis, datový typy budou `Int` a bude `init(duration: UInt)`, kde si to castneš na `Int`?

Reply:
mimochodem, podle dokumentace Realm neumí `UInt` uložit

Reply:
No nevim jestli se mi to libi :-D

Reply:
Tak jeste nevime jestli Realm...

Reply:
když už se děláme s Reduxem, tak tam můžem nacpat SQLite 😄

Reply:
Tak asi bych byl stejně pro to zaobalit tak nějak těma structama, nějakejma servicama, aby to bylo nějak jedno kde to je uložený